src/main/java/app/entities/unitconverter/WeightConverter.java

```diff
@@ -0,0 +1,34 @@
+package app.entities.unitconverter;
+
+public class WeightConverter
+{
+    public double kgToTon(double kg)
+        {
```

Comment:
this indentation is something that is usually addressed in these PRs.

We did however not settle on a code convention, but when we do this will not be it 😉